### DISCUSSION TITLE
Select first item of Options picker

### DISCUSSION THREAD

**Discussion by diogenesbrussels** [Budibase/budibase#8328] [Help] [answered]
Hello,

I have an option picker component whose values are filled in according to the user's profile (works well with a data provider).
I want to always select the first option in the picker, how can I achieve this? Could this be a default value option?

Thank you!
Olivier

**Comment by diogenesbrussels** [comment]
Tested as default value but not working:
{{itemAt action_form.Fields.project 0}}
{{first action_form.Fields.project 1}}

The real question seems to be: how can I access the values populated in an option picker?

**Comment by andz-bb** [comment, marked as answer]
Hey @diogenesbrussels 

Are you using a Data Provider as a source for your Options Picker? I was able to auto select the first option by setting the default value to the _id of the first item in my Data Provider result. I used this binding for my default value: `{{ first (pluck Data Provider.Rows '_id') }}`

The pluck returns an array with an _id from each object returned by the Data Provider, and then the first helper returns the first of those. Hope this is helpful for you!

**Reply by diogenesbrussels** [reply]
It seems obvious when I read your answer, sometimes we don't see the easiest way, thanks!

**Reply by diogenesbrussels** [reply]
Hello @andz-bb 

I was using this solution and it worked without a problem. Since I upgraded to version 2.2.3, the choice remains empty no matter what value is passed if it is from a handlebar or a js function (if it is text, no problem).
I tested the return of the function which is a string and with the right value. I also tested a js function to get the same result: 
`return Object.values($("projects_provider.Rows"))[0].name;`

Has there been a change in the way the default value is handled? Is this a regression?

Thanks a lot!
